frontend/include/chpl/resolution/scope-types.h

```diff
@@ -103,51 +110,28 @@ class OwnedIdsWithName {
   reference to a collection stored in OwnedIdsWithName.
  */
 class BorrowedIdsWithName {
- private:
-  // TODO: consider storing a variant of ID here
-  // with symbolPath, postOrderId, and tag
-  ID id_;
-  const std::vector<ID>* moreIds_ = nullptr;
- public:
-  /** Construct an empty BorrowedIdsWithName */
-  BorrowedIdsWithName() { }
-  /** Construct a BorrowedIdsWithName referring to one ID */
-  BorrowedIdsWithName(ID id) : id_(std::move(id)) { }
-  /** Construct a BorrowedIdsWithName referring to the same IDs
-      as the passed OwnedIdsWithName.
-      This BorrowedIdsWithName assumes that the OwnedIdsWithName
-      will continue to exist. */
-  BorrowedIdsWithName(const OwnedIdsWithName& o)
-    : id_(o.id_), moreIds_(o.moreIds_.get())
-  { }
+ // To allow us to use the private constructor
+ friend class OwnedIdsWithName;
 
-  /** Return the number of IDs stored here */
-  int numIds() const {
-    if (moreIds_ == nullptr) {
-      return 1;
-    }
-    return moreIds_->size();
+ private:
+  static inline bool isIDVisible(const OwnedID& id, bool skipPrivateVisibilities) {
```

Comment:
I would prefer `isIdVisible`, I think that's generally the convention we're going for (you'd only see `ID` if it's all on its own as a type name).